README.md

```diff
@@ -573,3 +574,157 @@ export class MyAwesomeComponent extends React.PureComponent {
     }
 }
 ```
+## External libraries
+
+### Prefer native js vs lodash
```

Comment:
a good section, i like it. my common feedback - please add few words regarding to polyfills. Otherwise it looks unsafely to swapping to native methods.

Reply:
Maybe we will leave only bad cases to not divide all lodash functionality to good or bad? I described popular and often used methods. In cases above, we can use Object.(keys|values|entries) and after use array methods.

Reply:
I guess we have to provide a list of all bad technics we have to replace to native alternatives. Regarding to using two methods vs one: using two methods vs one is not a good choice. We have to make our code simple and do not introduce extra boilerplate code. Lodash is created for this. Still need your input about polyfills and cross browser limitations.

Reply:
It's a rare case where you need reduce to object and I think it's nice to directly show where is a plain array or where is an object which you need to use as an array. You use one short method with particular cases and your code remains strict, native and without external modules except using external functionality which eats any type